HTCONDOR-1152: isolate GPUs with cgroups v1 (#2142)
htcondor/htcondor: docs/version-history/feature-versions-23-x.rst

New Features:

> - On Linux systems with cgroup v1 enabled, HTCondor now uses the device cgroup

Reviewer: ```suggestion
- On Linux systems with cgroup v1 enabled, HTCondor now uses the "devices" cgroup
```

>   to hide GPUs that are not provisioned by the system.  This can be disabled

Reviewer: ```suggestion
  to hide GPUs that it did not provision.  This can be disabled
```

Reviewer: We call them "AssignedGPUs" in the slot ad, and "the system" is either wrong or ambiguous.
